lib/levels.js

```diff
@@ -65,13 +69,14 @@ function setLevel (level) {
   if (values[level] === undefined) throw Error('unknown level ' + level)
   const preLevelVal = this[levelValSym]
   const levelVal = this[levelValSym] = values[level]
+  const useOnlyCustomLevelsVal = this[useOnlyCustomLevelsSym]
```

Comment:
have you verifyies this property is defined and of the correct value here?

Reply:
I had put a `console.log` in the code and for my test its `true` and for the other tests where it is actually defined. I'm just confused that the test actually returns `null` for priority. That's really what I don't understand.